velox/common/memory/tests/SharedArbitratorTest.cpp

```diff
@@ -2147,6 +2150,60 @@ DEBUG_ONLY_TEST_F(SharedArbitrationTest, driverInitTriggeredArbitration) {
       .assertResults(expectedVector);
 }
 
+DEBUG_ONLY_TEST_F(SharedArbitrationTest, reclaimerStats) {
+  std::vector<RowVectorPtr> vectors;
+  const int vectorSize = 100;
+  fuzzerOpts_.vectorSize = vectorSize;
+  vectors.push_back(newVector());
+
+  createDuckDbTable(vectors);
+  setupMemory(32 * MB, 0);
+  std::shared_ptr<core::QueryCtx> queryCtx = newQueryCtx(32 * MB);
+  ASSERT_EQ(queryCtx->pool()->capacity(), 0);
+  ASSERT_EQ(queryCtx->pool()->maxCapacity(), 32 * MB);
+  auto additionalCtxLeafPool = queryCtx->pool()->addLeafChild("ctx_leaf");
+  TestAllocation outerAlloc;
+  outerAlloc.buffer = additionalCtxLeafPool->allocate(8 * MB);
+  outerAlloc.pool = additionalCtxLeafPool.get();
+  outerAlloc.size = 8 * MB;
+  fakeOperatorFactory_->setAllocationCallback([&](Operator* op) {
+    TestAllocation allocation0;
```

Comment:
you might want to use an atomic bool to make sure we trigger arbitration reclaim once with multiple drivers?